pkg/network/port_test.go

```diff
@@ -155,24 +156,25 @@ func TestReadInitialUDPState(t *testing.T) {
 		require.NoError(t, err)
 		for _, p := range ports[:2] {
 			if _, ok := initialPorts[PortMapping{testRootNs, p}]; !ok {
-				t.Errorf("PortMapping(testRootNs, p) returned false for port %d", p)
+				t.Logf("PortMapping(testRootNs, p) returned false for port %d", p)
```

Comment:
Can we actually rewrite these bits using `assert.Contains`? Replacing with `Logf` will not fail the test eventually which is what we here.

Reply:
I decided for us to use `assert.EventuallyWithT` so we can freely use assertions and not deal with the boolean return.